resources/js/features/games/components/TopAchieversMainRoot/TopAchieversList/TopAchieversList.tsx

```diff
@@ -0,0 +1,85 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  BaseTable,
+  BaseTableBody,
+  BaseTableCell,
+  BaseTableHead,
+  BaseTableHeader,
+  BaseTableRow,
+} from '@/common/components/+vendor/BaseTable';
+import { PlayerBadgeIndicator } from '@/common/components/PlayerBadgeIndicator';
+import { PlayerBadgeLabel } from '@/common/components/PlayerBadgeLabel';
+import { UserAvatar } from '@/common/components/UserAvatar';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { AwardType } from '@/common/utils/generatedAppConstants';
+import { formatDate } from '@/common/utils/l10n/formatDate';
+import { cn } from '@/utils/cn';
+
+export const TopAchieversListContainerTestId = 'top-achievers-list';
+
+export const TopAchieversList: FC = () => {
+  const { paginatedUsers } = usePageProps<App.Platform.Data.GameTopAchieversPageProps>();
+
+  const { t } = useTranslation();
+
+  if (!paginatedUsers.items.length) {
+    return null;
+  }
+
+  return (
+    <BaseTable
+      containerClassName={cn(
+        'overflow-auto rounded-md border border-neutral-700/80 bg-embed',
+        'light:border-neutral-300 lg:overflow-visible lg:rounded-sm',
+        '',
+        '',
+      )}
+    >
+      <BaseTableHeader>
+        <BaseTableRow className="do-not-highlight">
+          <BaseTableHead>{t('Rank')}</BaseTableHead>
+          <BaseTableHead>{t('User')}</BaseTableHead>
+          <BaseTableHead>{t('Progress')}</BaseTableHead>
+        </BaseTableRow>
+      </BaseTableHeader>
+
+      <BaseTableBody>
+        {paginatedUsers.items.map((achiever) => (
+          <BaseTableRow key={achiever.user.displayName}>
+            <BaseTableCell>{achiever.rank}</BaseTableCell>
```

Comment:
This should pass through the number formatter, eg:

```tsx
const { formatNumber } = useFormatNumber();

<BaseTableCell>{formatNumber(achiever.rank)}</BaseTableCell>
```

otherwise number separators will not work correctly for any locale, ie:

![Screenshot 2024-11-23 at 6 37 00 PM](https://github.com/user-attachments/assets/293d119f-d5d7-4ca7-b696-0a2b3c4d62fa)

Reply:
done